### ISSUE TITLE
Firefox based browsers complains about invalid certificate for api.linuxcounter.net and for blog.linuxcounter.net

### ISSUE THREAD

**Issue by alexloehner** [alexloehner/linuxcounter.new#13]
"Iceweasel is crying about invalid domain for cert:

blog.linuxcounter.net uses an invalid security certificate. The certificate is only valid for the following names: www.linuxcounter.net, owa.linuxcounter.net, mail.linuxcounter.net, autodiscover.linuxcounter.net, linuxcounter.net (Error code: ssl_error_bad_cert_domain)

Same cert is used for api subdomain. What is interesting even adding exception did not help (but it is rather not your fault ;D)."

**Comment by alexloehner** [comment]
okay, I've found the problem and the solution.
The problem was a header my webserver sends with each request to
www.linuxcounter.net. Since this domain is forced to SSL, it always
sends the header "Strict-Transport-Security".
This header has several parameters and one of it is: "includesubdomains"

I've removed this part from the header and now this should not happen
anymore.

But there is one thing you now have to do to get rid of this problem
within YOUR browser:

Type in the address bar:  "about:permissions"

Then search for ALL occurences of linuxcounter.net, including all
subdomains and delete these entries (Button on the top right: "Forget
about this website").

After a browser restart, this problem is gone.
